cime_config/testdefs/testlist_clm.xml

```diff
@@ -2034,17 +2034,16 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="ERS_Ln9" grid="ne0ARCTICne30x4_ne0ARCTICne30x4_mt12" compset="I1850Clm60Sp" testmods="clm/clm60cam6LndTuningMode_1979Start">
+  <test name="ERS_Ln9" grid="ne0ARCTICne30x4_ne0ARCTICne30x4_mt12" compset="ISSP245Clm50BgcCrop" testmods="clm/clm50cam6LndTuningMode_1979Start">
```

Comment:
For our purposes, I changed three existing tests to transient because I found that their test descriptions already referred to transient cases, which these were not originally.

To use the only ISSP245 compset `ISSP245Clm50BgcCrop`, for all three tests I had to also point to the clm50... testmods directory rather than the clm60...

Reply:
The CAM test is...

``` xml
<test compset="FHIST" grid="ne0ARCTICne30x4_ne0ARCTICne30x4_mt12" name="SMS_D_Ln9_P1280x1" testmods="cam/outfrq9s" supported="false">
    <machines>
```

So I think we should do a Clm60Sp HIST compset for this grid. CAM is using Clm50 right now, so you could do that (but then we later need to change it), or also have a Clm50 test that we remove when CAM updates (#2379).

``` xml
<test name="SMS_Ln9" grid="ne0ARCTICne30x4_ne0ARCTICne30x4_mt12"  compset="IHistClm60Sp" testmods="clm/clm60cam6LndTuningMode_1979Start">
```

Reply:
@ekluzek we talked about also making a Clm51 version of this test, acknowledging that the two are identical; however, I get this error in setup_logic_lnd_tuning:
`First part of lnd_tuning_mode MUST match the CLM version you are using` 
...because we do not have a 5.1 equivalent of the clm60cam6LndTuningMode_1979Start testmods directory. To me it seems counterproductive to add new Clm51 testmods now. I would rather remove the test. Let me know if you have a different suggestion.

Reply:
I agree that makes the most sense...